modules/internal/ChapelTaskData.chpl

```diff
@@ -101,6 +102,26 @@ module ChapelTaskData {
     return v == 1;
   }
 
+  proc chpl_task_data_setNextCoStmtSerial(tls:c_ptr(chpl_task_infoChapel_t), makeSerial: bool) : void {
+    var prv = tls:c_ptr(c_uchar);
+    var i = chpl_offset_nextCoStmtSerial;
+    var v:uint(8) = 0;
+    if makeSerial then
+      v = 1;
+    c_memcpy(c_ptrTo(prv[i]), c_ptrTo(v), c_sizeof(uint(8)));
+  }
+
+  proc chpl_task_data_getNextCoStmtSerial(tls:c_ptr(chpl_task_infoChapel_t)) : bool {
+    var ret:bool = false;
+    var prv = tls:c_ptr(c_uchar);
+    var i = chpl_offset_nextCoStmtSerial;
+    var v:uint(8) = 0;
+    c_memcpy(c_ptrTo(v), c_ptrTo(prv[i]), c_sizeof(uint(8)));
+    if boundsChecking then
+      assert(v == 0 || v == 1);
+    return v == 1;
```

Comment:
For some reason I think it'd be better to write `return v != 0` but I'm not sure I can articulate why. (Maybe because of how `if (x)` works in C it seems more natural to me).

Reply:
I copied `chpl_task_data_getSerial` here too.